test/integration/full-content/fixtures/core__rss.serialized.html

```diff
@@ -0,0 +1 @@
+<!-- wp:rss /-->
```

Comment:
I'm personally very surprised seeing that it doesn't contain any metadata in HTML comment. It looks like it's the same case for other blocks like Latest Posts. I will investigate it separately.

Reply:
> I'm personally very surprised seeing that it doesn't contain any metadata in HTML comment. It looks like it's the same case for other blocks like Latest Posts. I will investigate it separately.

It is surprising, as it would imply that given the input of `core__rss.html`, the editor would destroy all attributes in a reserialization.

There are also a number of attributes defined in `core__rss.html` which simply don't exist for the defined block type (e.g. `postsToShow`), perhaps part of earlier iterations of the block.

Fix proposed at #14122